packages/app/src/cli/utilities/developer-platform-client/app-management-client.test.ts

```diff
@@ -113,3 +151,37 @@ describe('allowedTemplates', () => {
     expect(got).toEqual([templateWithoutRules, allowedTemplate])
   })
 })
+
+describe('devStoresForOrg', () => {
+  test('returns a list of OrganizationStore typed stores, from query response', async () => {
+    const mockStore1 = LIST_DEV_STORE_RESPONSE_VALUE.organization.properties.edges[0]?.node
+    const mockStore2 = LIST_DEV_STORE_RESPONSE_VALUE.organization.properties.edges[1]?.node
+
+    const mockAppManagementClient = new AppManagementClient()
+    vi.mocked(businessPlatformOrganizationsRequest).mockResolvedValue(LIST_DEV_STORE_RESPONSE_VALUE)
+
```

Comment:
Getting a `mockResolvedValue` function doesn't exist error, though not sure why?

Reply:
Turns out I have to include the mocked files, ie. `vi.mock('@shopify/cli-kit/node/api/business-platform')`